src/components/Section/History/Gap.jsx

```diff
@@ -1,42 +1,54 @@
+/* eslint-disable react/no-array-index-key */
 import React from 'react'
-import { i18n } from '../../../config'
-import { DateSummary } from '../../Summary'
+import PropTypes from 'prop-types'
+
+import { DateSummary } from 'components/Summary'
 
 /**
  * Renders a formatted gap row
  */
-export class Gap extends React.Component {
-  render() {
-    if (!this.props.dates.from || !this.props.dates.to) {
-      return null
-    }
-
-    const dates = DateSummary(this.props.dates)
-    return (
-      <div className="gap details open">
-        <div className="usa-alert usa-alert-error" role="alert">
-          <div className="usa-alert-body">
-            <h5 className="usa-alert-heading">{this.props.title} - <span className="dates">{dates}</span></h5>
-
-            <p>{this.props.para}</p>
-            <button className="usa-button-outline" onClick={this.props.onClick}>
```

Comment:
It looks like this didn't make it over to the new component. Is that intentional?

Reply:
Actually never mind. I understand. There isn't an actual "gap" item anymore. It's just an error message that informs of a gap.